### PR TITLE
178 refactor filter attributes

### LINKED ISSUE #178: [Bug] filter_attributes function is located in an unexpected location
### Description

The `filter_attributes()` function, located here: https://github.com/code42/py42/blob/master/src/py42/sdk/queries/query_filter.py#L188 does not make a lot of sense.  It is used on constants all around py42.  It should be moved to `util.py` or somewhere better.

### Steps to Reproduce

1. Follow link and see that the function is in a weird spot

### Expected Behavior
This be importable from util.py

### Actual Behavior
Have to import from query_filter.py

### PR DESCRIPTION
**Pull request by DiscoRiver**
### Description of Change

- Refactored `filter_attributes` to `get_attribute_keys_from_class`, and moved from query_filter.py to util.py.

### Issues Resolved

- Fixes #178

### Testing Procedure

- Import using `from py42.util import get_attribute_keys_from_class` instead of `from py42.sdk.queries.query_filter import get_attribute_keys_from_class`

### CODE REVIEW

**Comment by DiscoRiver** [comment]
I forgot to pull in the refactoring from `master` before I started working on this fix, so it was easier to create a new branch and do it on the new code instead of trying to fix the previous.

File: CHANGELOG.md [hunk 1]
```diff
@@ -103,6 +103,7 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ### Changed
 
+- `filter_attributes` now imports as `` from `py42.util` instead of `py42.sdk.queries.query_filter`.
```
Comment (unparalleled-js) [inline CHANGELOG.md:106]: Still implies it is called `filter_attributes` here.  Could do:
`py42.sdk.queries.query_filter` renamed to `py42.util.get_attribute_keys_from_class`.

File: src/py42/util.py [hunk 2]
```diff
@@ -72,3 +72,19 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def get_attribute_keys_from_class(cls):
+    """Returns attribute names for the given class object
```
Comment (unparalleled-js) [inline src/py42/util.py:78]: I don't think you need the word `object` here.  That kind of implies an instance is required.

**Comment by unparalleled-js** [comment]
Looking good! Just a couple of documentation improvements would make it perfect

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**